### PR TITLE
Skip the failing inabox BTF test on Safari.

### PR DESCRIPTION
**Pull request by zombifier**
See #21311.

### CODE REVIEW

**Comment by zombifier** [comment]
@lannka I still haven't figured out why the tests fail on Saucelabs, so feel free to merge this PR to unbreak it for now.

**Comment by zombifier** [comment]
@danielrozenberg you may want to look at this. I left the tests running for a week and it still isn't fixed.

**Comment by danielrozenberg** [comment]
Not sure what you mean to ask of me here. If you can't get these tests to pass, you can disable them - we can't merge a PR that will turn `master` red

**Comment by zombifier** [comment]
Sorry; this PR *does* disable them.

**Comment by danielrozenberg** [comment]
But the Travis build is still red, this test fails the single-pass shard... /cc @erwinmombay

**Comment by zombifier** [comment]
Can you rerun Travis? Must be a flake, as this PR does nothing but disable tests.

**Comment by danielrozenberg** [comment]
This passes now. I believe @lannka is OOO, but maybe @zhouyx can take a look and merge?

**Comment by zhouyx** [comment]
@zombifier Thank you for sending the PR. Sorry for the late response. I think I don't understand why are we trying to disable the test. Is it failing right now?

**Comment by zombifier** [comment]
Yes, I believe they're responsible for SauceLabs tests timing out.

**Comment by calebcordry** [comment]
@zombifier I am taking this over since now @zhouyx is OOO as well. It does seem like these tests are problematic. Would you mind creating a new issue to track why they are broken and adding a todo? Thanks

**Comment by zombifier** [comment]
@calebcordry Thanks. I have indeed made #21311 for tracking. Adding a comment in a minute.

File: test/integration/test-amp-inabox.js [hunk 1]
```diff
@@ -212,6 +212,7 @@ describe.configure().skipWindows().run('inabox', function() {
     });
   });
 
+  // TODO(zombifier, #21311): Investigate why Safari on SauceLabs timeout.
```
Comment (rsimha) [inline test/integration/test-amp-inabox.js]: @zombifier This test occasionally times out on Chrome too: https://travis-ci.org/ampproject/amphtml/jobs/524635005#L1527

Comment (rsimha) [reply test/integration/test-amp-inabox.js]: And for single-pass builds: https://travis-ci.org/ampproject/amphtml/jobs/524681807#L1908-L1912

**Pull request merged**